alt/content.md

```diff
@@ -0,0 +1,15 @@
+# ALT Linux
+
+This image serves as the `official ALT image` for the [ALT Linux](https://altlinux.org/).
+
+%%LOGO%%
+
+# About this image
+
+The `%%IMAGE%%:latest` tag will always point the latest stable release (which is, at the time of this writing, `%%IMAGE%%:p8`).
+
+## How It's Made
+
+The rootfs tarballs for this image are built using the [mkimage](https://en.altlinux.org/Mkimage) and [hasher](https://en.altlinux.org/Hasher), with an explicit goal being that they are transparent and reproducible. Using the same toolchain, it should be possible to regenerate (clean-room!) the same tarballs used for building the official Debian images.
```

Comment:
Is this paragraph accurate? (especially the part about "the official Debian images" ? :smirk:)

Just to be clear, are the `mkimage` and `hasher` results deterministically reproducible (bit-for-bit)?  That's the reason the Debian image included this bit -- those tarballs can be bit-for-bit reproduced.

Reply:
Oh, sorry. Wrong copy-paste. I will fix it on next commit.